File: ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { IpProxyDetectionSDK } from '..'


describe('exists', async () => {

  test('test-mode', () => {
    const testsdk = IpProxyDetectionSDK.test()
    equal(testsdk instanceof IpProxyDetectionSDK, true,
      'IpProxyDetectionSDK.test() must return a client synchronously')
  })

})

File: lua/test/exists_test.lua
-- ProjectName SDK exists test

local sdk = require("ip-proxy-detection_sdk")

describe("IpProxyDetectionSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
